{"name": "Ручка /api/v1/courier", "status": "passed", "description": "Создаём курьера, затем создаём ещё одного курьера, с такими же данными. Должна быть неудача.", "start": 1710617468983, "stop": 1710617469103, "uuid": "5bf31943-1457-4dec-ba68-6c3e4d398a16", "historyId": "2c26b9034b3d161dd45c110f60ac5882", "testCaseId": "2c26b9034b3d161dd45c110f60ac5882", "fullName": "test_api.tests.test_create_courier.TestCreateCourier#test_create_simular_couriers_error_409", "labels": [{"name": "parentSuite", "value": "test_api.tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "LAPTOP-KS9V29SF"}, {"name": "thread", "value": "5592-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_api.tests.test_create_courier"}]}